components/02-molecules/banner/grand-hero/yds-grand-hero.twig

```diff
@@ -0,0 +1,58 @@
+{#
+ # Available Props:
+ # - grand_hero__content__background
+ # - grand_hero__overlay_variation: contained (default), full
+ #
+ # Available Variables:
+ # - grand_hero__heading
+ # - grand_hero__snippet
+ # - grand_hero__link__content
+ # - grand_hero__link__url
+ #
+ # Available Blocks:
+ # - grand_hero__media
+ #}
+
+{% set grand_hero__base_class = 'grand-hero-banner' %}
+{% set grand_hero__content__background = grand_hero__content__background|default('basic-black') %}
```

Comment:
`basic-black` isn't one of our color schemes. If you go to the color pairings page (http://localhost:6006/?path=/story/tokens-colors--color-pairings) the darkest one we have is `gray-800`. You might ask Ashley if that will work. Otherwise, we'll need to add a new color scheme.

Actually... I seem to remember her mentioning that this color might be a "custom" color chosen by the theme... if that's the case, we'll need a little more details on what colors can be chosen etc, because we might not even use our `component-theme` thing for this...